### DISCUSSION TITLE
ScriptReference/Networking.DownloadHandlerAudioClip-streamAudio

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#569] [Notes]
# ScriptReference/Networking.DownloadHandlerAudioClip-streamAudio

https://docs.unity3d.com/ScriptReference/Networking.DownloadHandlerAudioClip-streamAudio.html

**Comment by ForemanDev** [comment]
Streaming audio is broken. :( On a WebGL build, Unity 2021.3.21f1, I get the error:

`Playback of audio clip not yet possible; headers are done, 207516/? (66.09%) bytes downloaded but size is still not known
Error: Cannot create FMOD::Sound instance for clip "" (FMOD error: The HTTP request timed out. )`
